SmartDeviceLink/SDLNotificationConstants.h

```diff
@@ -42,6 +39,26 @@ typedef void (^SDLRPCNotificationHandler)(__kindof SDLRPCNotification *notificat
  */
 typedef void (^SDLResponseHandler)(__kindof SDLRPCRequest *__nullable request, __kindof SDLRPCResponse *__nullable response, NSError *__nullable error);
 
+
+/**
+ A handler that may optionally be run when an SDLSubscribeButton or SDLSoftButton has a corresponding notification occur.
+
+ @warning This only works if you send the RPC using SDLManager.
+ @warning Only one of the two parameters will be set for each block call.
+
+ @param buttonPressNotification An SDLOnButtonPress object that corresponds to this particular button.
+ @param buttonEventNotification An SDLOnButtonEvent object that corresponds to this particular button.
+ */
+typedef void (^SDLRPCButtonNotificationHandler)(SDLOnButtonPress *_Nullable buttonPressNotification,  SDLOnButtonEvent *_Nullable buttonEventNotification);
```

Comment:
can we shorten these to `buttonPress` and `buttonEvent`? We don't really care if it is a notification, really.